draft-ietf-httpbis-priority.md

```diff
@@ -541,12 +541,19 @@ background urgency being associated. However, in the worst case, the asymmetry
 between the precedence declared by multiple clients might cause responses going
 to one end client to be delayed totally after those going to another.
 
-In order to mitigate this fairness problem, when a server responds to a request
-that is known to have come through an intermediary, the server SHOULD prioritize
-the response as if it was assigned the priority of  `u=1, i`
-(i.e. round-robin) regardless of the value of the Priority header field being
-transmitted, unless the server knows the intermediary is not
-coalescing requests from multiple clients.
+In order to mitigate this fairness problem, a server could use knowledge about
+the intermediary as another signal in its prioritization decisions. For
+instance, if a server knows the intermediary is coalescing requests, then
+it could prioritize responses using a fixed urgency level with incremental
+loading. Treating all requests from an intermediary in this way would result in
+round-robin scheduling. This works well when the network bottleneck exists
+between the intermediary and the end client, as the intermediary buffers
```

Comment:
This assumes that the network is always the bottleneck.  Sometimes it is processing resources that are being prioritized.

> This can work if the constrained resource is network capacity between the intermediary and user agent.

(End client == user agent?)